tensorflow/core/kernels/matmul_op_test.cc

```diff
@@ -281,21 +286,23 @@ TYPED_TEST_SUITE_P(FusedMatMulWithBiasOpTest);
 // -------------------------------------------------------------------------- //
 
 TYPED_TEST_P(FusedMatMulWithBiasOpTest, MatMul256x256x256) {
-  this->VerifyMatMulWithBias(256, 256, 256, false, false);
-  this->VerifyMatMulWithBias(256, 256, 256, true, false);
-  this->VerifyMatMulWithBias(256, 256, 256, false, true);
-  this->VerifyMatMulWithBias(256, 256, 256, true, true);
+  this->VerifyMatMulWithBias(256, 128, 64, false, false);
```

Comment:
Why the change from 256 x 256 to 128 x 64?
Also, if this is necessary, then please also rename the test name accordingly.

Reply:
yes, I need to check it out with Eugene when he is available, it might be that this is only necessary for the ROCm platform

Reply:
According to Eugene, this was done in order to make sure that the matrix dimensions are handled in a correct order. Otherwise, if one accidentally permutes 256x256x256, this won't be caught by the test, I will rename the test(s) accordingly